core/src/test/java/org/apache/calcite/test/RelMdColumnOriginsTest.java

```diff
@@ -74,4 +78,25 @@
     statement.close();
     connection.close();
   }
+
```

Comment:
Extending from `SqlToRelConverterTest` is not a good idea, why not move the test case directly into `RelOptRulesTest` ?

Reply:
ok, I will try it ,thanks! I thought i was testing RelMdColumnOrigins, so ...

> Extending from `SqlToRelConverterTest` is not a good idea, why not move the test case directly into `RelOptRulesTest` ?